book/moving_around.md

```diff
@@ -113,29 +125,78 @@ The quoting techniques above are useful when constructing glob-literals, but you
    ls $glob_pattern
    ```
 
+## Creating a Directory
+
+As with most other shells, the [`mkdir` command](/commands/docs/mkdir.md) is used to create a new directory. However, unlike some other `mkdir` implementations, Nushell's version:
+
+- Will create multiple directory levels automatically. For example:
+
+  ```nu
+  mkdir modules/my/new_module
+  ```
+
+  This will create all three directories even if none of them currently exists. On Linux/Unix, this requires `mkdir -p`.
+
+  ::: tip
+  A common mistake when coming to Nushell is to attempt to use `mkdir -p <directory>` as in the native Linux/Unix version. However, this will generate an `Unknown Flag` error on Nushell.
+
+  Just repeat the command without the `-p` to achieve the same effect.
+
+  ```
+
+  ```
+
+- Will not error if the directory already exists. For example:
+
+  ```nu
+  mkdir modules/my/new_module
+  mkdir modules/my/new_module
+  # => No error
+  ```
+
+These features greatly simplify directory creation in scripting.
```

Comment:
massive nitpick, but maybe "greatly" is overselling it :sweat_smile:

Reply:
Ah yeah, I forgot that `mkdir -p` doesn't error if the directory exists either.  I need to rework this section entirely.